fail2ban/server/utils.py

```diff
@@ -116,7 +117,22 @@ def setFBlockMode(fhandle, value):
 		return flags
 
 	@staticmethod
-	def executeCmd(realCmd, timeout=60, shell=True, output=False, tout_kill_tree=True, success_codes=(0,)):
+	def buildShellCmd(realCmd, varsDict):
```

Comment:
??
No matter what for characters the value has... It will be appended as argument in the command-tuple (array).
E. g. from 
```python
'echo "failure of <f-user>: <matches>"'
``` 
will be to:
```python
['f2bV_matches=$0 f2bV_matches=$1\necho "failure of $f2bV_f-user: f2bV_matches"', 'unknown user', 'this is the string from tag matches']
```